core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -52,11 +37,6 @@ describe('Learner dashboard functionality', function() {
   beforeAll(function() {
     libraryPage = new LibraryPage.LibraryPage();
     learnerDashboardPage = new LearnerDashboardPage.LearnerDashboardPage();
-    collectionEditorPage = new CollectionEditorPage.CollectionEditorPage();
-    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
-    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
-    explorationEditorMainTab = explorationEditorPage.getMainTab();
-    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
```

Comment:
@nithusha21 Are these expected changes? Please check this and other test files. Just want to confirm before making any further changes in other files. Thanks.

Reply:
If these are indeed unused variables, then yes, I think it should be fine. My suggestion would be to try to run the test suite after you make the changes, and make sure it works